cscs-checks/apps/amber/amber_check.py

```diff
@@ -57,65 +54,35 @@
     },
 }
 
+REFERENCE_ENERGY = {
+    # every system has a different reference energy and drift
+    'Cellulose_production_NVE': (-443246.0, 5.0E-05),
+    'FactorIX_production_NVE': (-234188.0, 1.0E-04),
+    'JAC_production_NVE_4fs': (-44810.0, 1.0E-03),
+    'JAC_production_NVE': (-58138.0, 5.0E-04),
+}
 
-class AmberBaseCheck(rfm.RunOnlyRegressionTest):
-    valid_prog_environs = ['builtin']
-    strict_check = False
-    modules = ['Amber']
-    extra_resources = {
-        'switches': {
-            'num_switches': 1
-        }
-    }
-    maintainers = ['VH', 'SO']
-    tags = {'scs', 'external-resources'}
 
+class AmberCheck(AmberBaseCheck):
+    modules = ['Amber']
     benchmark = parameter([
         # NVE simulations
         'Cellulose_production_NVE',
         'FactorIX_production_NVE',
         'JAC_production_NVE_4fs',
         'JAC_production_NVE',
     ])
-
-    @run_after('init')
-    def download_files(self):
-        self.prerun_cmds = [
-            # cannot use wget because it is not installed on eiger
-            f'curl -LJO https://github.com/victorusu/amber_benchmark_suite'
-            f'/raw/main/amber_16_benchmark_suite/PME/{self.benchmark}.tar.bz2',
-            f'tar xf {self.benchmark}.tar.bz2'
-        ]
-
-    @run_after('init')
-    def set_energy_and_tolerance_reference(self):
-        self.ener_ref = {
-            # every system has a different reference energy and drift
-            'Cellulose_production_NVE': (-443246, 5.0E-05),
-            'FactorIX_production_NVE': (-234188, 1.0E-04),
-            'JAC_production_NVE_4fs': (-44810, 1.0E-03),
-            'JAC_production_NVE': (-58138, 5.0E-04),
+    strict_check = False
+    extra_resources = {
+        'switches': {
+            'num_switches': 1
         }
+    }
+    references = REFERENCE_ENERGY
```

Comment:
You don't need to store this here since it's not used. Also, notice that we have already a variable `reference`, so having another one named as `references` is simply a no.